oprheum-game/SnowballScene.swift

```diff
@@ -66,16 +69,18 @@ class SnowballScene: BaseScene {
 
     func showWinOverlay() {
         timerValue.alpha = 0
+
+        let showWinOverlayAction = SKAction.fadeInWithDuration(0.3)
+        winOverlay.runAction(showWinOverlayAction)
         winOverlay.show()
-        let hideWinSceneAction = SKAction.fadeOutWithDuration(0.3)
-        gameObjects.runAction(hideWinSceneAction)
```

Comment:
It was part of the merge (of what I had + master)... does the order matter/where should it go?